<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableTransformWidget</class>
 <widget class="QWidget" name="TableTransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table Transform</string>
  </property>
  <layout class="QGridLayout" name="transform_layout">
   <item row="0" column="0">
    <widget class="QLabel" name="transform_type_label">
     <property name="text">
      <string>Transform:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="transform_type_combo">
     <item>
      <property name="text">
       <string>PCA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="transform_center_checkbox">
     <property name="text">
      <string>Center</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="transform_standardize_checkbox">
     <property name="text">
      <string>Standardize (z-score)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="transform_include_label">
     <property name="text">
      <string>Include columns:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="transform_include_edit">
     <property name="placeholderText">
      <string>Comma-separated names (optional)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="transform_exclude_label">
     <property name="text">
      <string>Exclude columns:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="transform_exclude_edit">
     <property name="placeholderText">
      <string>Comma-separated names (optional)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="transform_output_name_label">
     <property name="text">
      <string>Output Name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="transform_output_name_edit">
     <property name="placeholderText">
      <string>e.g., Base (PCA)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="apply_transform_btn">
     <property name="text">
      <string>Apply Transform</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</ui>

